src/bin/pg_autoctl/nodestate_utils.c

```diff
@@ -139,17 +145,24 @@ nodestateAdjustHeaders(NodeAddressHeaders *headers,
 		}
 	}
 
+	/*
+	 * In order to have a static nice table output even when using
+	 * auto-refreshing commands such as `watch(1)` when states are changing, we
+	 * always use the max known state length.
+	 */
+	headers->maxStateSize = MAX_NODE_STATE_LEN;
+
 	/* initialize to mininum values, if needed */
 	if (headers->maxNameSize == 0)
 	{
 		/* Name */
-		headers->maxNameSize = 5;
+		headers->maxNameSize = strlen("Name");
```

Comment:
I like this self documenting code

Reply:
I just grew fed-up of having to review and maintain those numbers everywhere...